src/components/UseV2ExchangeModal/index.tsx

```diff
@@ -29,42 +29,46 @@ const StyledButton = styled(Button)`
 
 type UseV2ExchangeModalProps = {
   onDismiss?: () => void
+  isPageVisit?: boolean
 }
 
-const UseV2ExchangeModal = ({ onDismiss = defaultOnDismiss }: UseV2ExchangeModalProps) => {
+const UseV2ExchangeModal = ({ onDismiss = defaultOnDismiss, isPageVisit = false }: UseV2ExchangeModalProps) => {
   const [isAcknowledged, setIsAcknowledged] = useState(false)
   const [hasTimerPassed, setHasTimerPassed] = useState(false)
   const [timerSecondsRemaining, setTimerSecondsRemaining] = useState(10)
 
   useEffect(() => {
-    const tick = () => {
-      setTimerSecondsRemaining((prevSeconds) => prevSeconds - 1)
-    }
-
-    const timerInterval = setInterval(() => tick(), 1000)
-
-    const preventClickHandler = (e) => {
-      e.stopPropagation()
-      e.preventDefault()
-      return false
-    }
+    if (isPageVisit) {
```

Comment:
Only lock modal for 10s if it is the pageVisitModal, i.e. 

<img width="423" alt="Screenshot 2021-05-12 at 11 29 56" src="https://user-images.githubusercontent.com/79279477/117960891-65e31c80-b315-11eb-880a-e9ed3a692e30.png">